src/ontology/mondo-ingest.Makefile

```diff
@@ -163,6 +163,21 @@ $(COMPONENTSDIR)/icd10who.owl: $(TMPDIR)/icd10who_relevant_signature.txt | compo
 		remove -T config/properties.txt --select complement --select properties --trim true \
 		annotate --ontology-iri $(URIBASE)/mondo/sources/icd10who.owl --version-iri $(URIBASE)/mondo/sources/$(TODAY)/icd10who.owl -o $@; fi
 
+# TODO: do
+# TODO: remove any unnecessary queries / lines here
+#  - fix_hgnc_mappings
+#  - maybe make a comment about other ones i thought to delete
+$(COMPONENTSDIR)/icd11foundation.owl: $(TMPDIR)/icd11foundation_relevant_signature.txt | component-download-icd11foundation.owl
+	if [ $(COMP) = true ] ; then $(ROBOT) remove -i $(TMPDIR)/component-download-icd11foundation.owl.owl --select imports \
+		rename --mappings config/property-map-1.sssom.tsv --allow-missing-entities true \
+		remove -T $(TMPDIR)/icd11foundation_relevant_signature.txt --select complement --select "classes individuals" --trim false \
+		remove -T $(TMPDIR)/icd11foundation_relevant_signature.txt --select individuals \
+		query \
+			--update ../sparql/fix-labels-with-brackets.ru \
+			--update ../sparql/fix_complex_reification.ru \
```

Comment:
## Question: `fix_complex_reification.ru`
Does this file only apply to ORDO?

<details><summary>Details & suggestions</summary>
<p>

I've since removed this line from the `icd11foundation` goal, but it's still on several other goals.

I ask because it at the end, it has:
```sparql
  FILTER (isIRI(?entity) && (regex(str(?entity), "http://www.orpha.net/ORDO/")))
  BIND(IF(STRSTARTS(str(?precision_value), "NTBT"), skos:narrowMatch, IF(STRSTARTS(str(?precision_value), "BTNT"), skos:broadMatch, oboInOwl:hasDbXref)) as ?property)
```

### Suggestions
1. If it indeed only applies to ORDO, then I suggest we remove this line from all the goals except for ORDO, as done in PR with similar situation: #459.
2. If that's the case, I suggest we also prepend `ordo_` to the filename (like I've suggest in similar situation https://github.com/monarch-initiative/mondo-ingest/pull/459#discussion_r1510414581).

</p>
</details>

Reply:
Resolving as out of scope for this PR. See: 
- #469